internal/pkg/cli/run_local.go

```diff
@@ -151,6 +161,32 @@ func newRunLocalOpts(vars runLocalVars) (*runLocalOpts, error) {
 		}
 		repoName := clideploy.RepoName(o.appName, o.wkldName)
 		o.repository = repository.NewWithURI(ecr.New(defaultSessEnvRegion), repoName, resources.RepositoryURLs[o.wkldName])
+
+		idPrefix := fmt.Sprintf("%s-%s-%s-", opts.appName, opts.envName, opts.wkldName)
```

Comment:
do we need the `o *runLocalOpts` parameter to `configureClients`, especially when `opts` is a pointer 🤔💭 ?